test/components/plots/areaPlotTests.ts

```diff
@@ -17,6 +17,24 @@ describe("Plots", () => {
       assert.strictEqual(plot.height(), 400, "was allocated height");
       svg.remove();
     });
+
+    it("adds a padding exception to the y scale at the constant y0 value", () => {
+      var svg = TestMethods.generateSVG(400, 400);
+      var xScale = new Plottable.Scales.Linear();
+      var yScale = new Plottable.Scales.Linear();
+      var constantY0 = 30;
+      yScale.addExtentsProvider((scale: Plottable.Scales.Linear) => [[constantY0, constantY0 + 10]]);
+      yScale.autoDomain();
+      assert.operator(yScale.domain()[0], "<", constantY0, "lower end of y Scale's domain is normally padded");
```

Comment:
Can we cut down on the redundancy?  Still unsure why anything is tested before the plot is instantiated.  You can retrieve the previous value and then compare it with a value after the plot is instantiated / rendered and then test that